CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.27.1] - 2022-02-XX
```

Comment:
I am a bit behind catching on the actual release process and the naming convention in this CHANGELOG so tagging Trung to check once. @ltrung could you please check this one?

Reply:
Rest looks good to me.

Reply:
Could you fix the date here? So instead of 2022-02-XX, it should be the day the next version open for development which would be todays as this will get in today :)

Reply:
Changed it to 2022-02-17